app.py

```diff
@@ -19,7 +19,7 @@ def __init__(self):
     def _appmetadata(self):
         pass
 
-    def _annotate(self, mmif: Union[str, dict, Mmif], **parameters) -> Mmif:
+    def _annotate(self, mmif: Union[str, dict, Mmif], whisper_language: str = "None", **parameters) -> Mmif:
```

Comment:
The language parameter (which wrongly named as `whisper_langauge` here) should be an element in the `parameters` kwargs, hence shouldn't be explicitly exposed in the method signature.